backend/.eslintrc.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
     "@typescript-eslint/no-empty-function": "off",
     "@typescript-eslint/no-unsafe-enum-comparison": "off",
     "no-void": "off",
+    "no-await-in-loop": "off",
```

Comment:
I wouldn't turn off this rule. Instead I would write for loops like this:

```js
for await (const item of items) {
   await something(item);
}
```

This won't give lint errors

Reply:
sweet, let me change it